src/transit_equity/geospatial/hex_to_shapely.py

```diff
@@ -0,0 +1,21 @@
+import geopandas as gpd
```

Comment:
Would it be preferable to have a more general name to this so that we can add more methods to this file?

Reply:
Sorry, I meant adding a more general name to the file instead of 'hex_to_shapely'

Reply:
done!